sumo-function-utils/lib/datatransformer.js

```diff
@@ -10,55 +10,91 @@
 function Transformer() {
 }
 
-Transformer.prototype.azureAudit = function (data) {
+Transformer.prototype.azureAudit = function (context, data) {
 
     var finalResult = [];
     if (data instanceof Array) {
-        data.forEach( message => {
+        data.forEach(message => {
             if (message.records instanceof Array) {
                 message.records.forEach(subMsg => {
-                    finalResult.push(subMsg);
+                    finalResult.push(this.generateFormattedLog(context, subMsg));
                 });
             } else {
-                finalResult.push(message);
+                finalResult.push(this.generateFormattedLog(context, message));
             }
         });
     } else {
         if (data.records instanceof Array) {
-            data.records.forEach(subMsg => { finalResult.push(subMsg);});
-        } else { finalResult.push(data);}
+            data.records.forEach(subMsg => { finalResult.push(this.generateFormattedLog(context, subMsg)); });
+        } else { finalResult.push(this.generateFormattedLog(context, data)); }
     }
     return finalResult;
 };
 
+/**
+ * Function to generate a well formatted log to send to Sumo, following GCM's standards
+ * @param msg the original Azure log message
+ * @returns {object} of log following GCM's standard format
+ */
+Transformer.prototype.generateFormattedLog = function (context, msg) {
```

Comment:
looks great: please triple check conformity with existing logs

Reply:
checked with cfn-deploy and some other repos, I don't think there is much more to add here